cli/args/lockfile.rs

```diff
@@ -147,22 +147,28 @@ impl CliLockfile {
       },
     };
 
+    let root_folder = workspace.root_folder_configs();
+    // CLI flag takes precedence over the config
+    let frozen = flags.frozen_lockfile.unwrap_or_else(|| {
+      root_folder
+        .deno_json
+        .as_ref()
+        .and_then(|c| c.to_lock_config().ok().flatten().map(|c| c.frozen()))
```

Comment:
Updated to handle it, but I'm noticing the interactions with the LSP are a bit weird. 

Right now the error is from the CLI (instead of a lint), so it just ends up in the LSP output instead of being a proper diagnostics. So you wouldn't get a proper error message until you ran a CLI command.

Reply:
I think that's fine. It just shows up for caching, right?